src/Microsoft.Health.Dicom.Client/DicomWebClient.ExtendedQueryTag.cs

```diff
@@ -91,5 +91,25 @@ public async Task<DicomWebResponse<IEnumerable<ExtendedQueryTagError>>> GetExten
             await EnsureSuccessStatusCodeAsync(response).ConfigureAwait(false);
             return new DicomWebResponse<IEnumerable<ExtendedQueryTagError>>(response, ValueFactory<IEnumerable<ExtendedQueryTagError>>);
         }
+
+        public async Task<DicomWebResponse<GetExtendedQueryTagEntry>> UpdateExtendedQueryTagAsync(string tagPath, UpdateExtendedQueryTagEntry newValue, CancellationToken cancellationToken)
+        {
+            EnsureArg.IsNotNullOrWhiteSpace(tagPath, nameof(tagPath));
+            EnsureArg.IsNotNull(newValue, nameof(newValue));
+            EnsureArg.EnumIsDefined(newValue.QueryStatus, nameof(newValue.QueryStatus));
```

Comment:
nit: name here probably should be `newValue` to properly format the error: `EnsureArg.EnumIsDefined(newValue.QueryStatus, nameof(newValue));`